ocw/lib/gce.py

```diff
@@ -11,7 +12,11 @@
 
 class GCE(Provider):
     __instances = {}
-    __skip_networks = frozenset({"default"})
+
+    if PCWConfig.has('cleanup/gce-skip-networks'):
```

Comment:
`__skip_networks = frozenset(ConfigFile().getList('cleanup/gce-skip-networks', "default"))`  - will replace whole if/else block

Reply:
No. The `.getList()` does not have 'default value'.

Reply:
https://github.com/SUSE/pcw/blob/master/webui/PCWConfig.py#L47

```
def getList(self, config_path: str, default: list = None) -> list:
        if default is None:
            default = []
        return [i.strip() for i in self.get(config_path, ','.join(default)).split(',')]
```

and what about second parameter named `default` ?